flashinfer/fused_moe/api.py

```
@@ -226,44 +231,63 @@ def __repr__(self) -> str:
+        variant: QuantVariant = QuantVariant.NVFP4,
         num_local_experts: int,
         hidden_size: int,
         intermediate_size: int,
         device=None,
         permute_cache=None,
     ):
-        """Build the ``trtllm_fp4_routed`` weight view from canonical bf16 weights.
+        """Build a ``trtllm_fp4_routed`` weight view from canonical BF16 weights.
 
         Register the result with ``MoEWeightPack.prepare_for("trtllm_fp4_routed", ...)``.
+        ``variant`` selects NVFP4, MXFP4×MXFP8, or ``QuantVariant.W4A16``
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)